packages/@functionless/aws-dynamodb-constructs/tsconfig.json

```diff
@@ -0,0 +1,34 @@
+{
+  "extends": "../../../tsconfig-base.json",
+  "compilerOptions": {
+    "outDir": "lib",
+    "declaration": true,
```

Comment:
nit: isn't declaration coming from base?

Reply:
Yeah, but i didn't bother to try and change things. We can fix it up later.